src/NServiceBus.Core/Licensing/LicenseSources.cs

```diff
@@ -21,30 +21,17 @@ public static LicenseSource[] GetLicenseSources(string licenseText, string licen
                 sources.Add(new LicenseSourceFilePath(licenseFilePath));
             }
 
-            sources.Add(new LicenseSourceConfigFile());
+            sources.Add(new LicenseSourceAppConfigLicenseSetting());
+            sources.Add(new LicenseSourceAppConfigLicensePathSetting());
 
-            sources.Add(new LicenseSourceFilePath(Path.Combine(AppDomain.CurrentDomain.BaseDirectory, "NServiceBus", "License.xml")));
-            sources.Add(new LicenseSourceFilePath(Path.Combine(AppDomain.CurrentDomain.BaseDirectory, "License", "License.xml")));
+            sources.Add(new LicenseSourceFilePath(Path.Combine(Environment.GetFolderPath(Environment.SpecialFolder.CommonApplicationData), "ParticularSoftware", "license.xml")));
+            sources.Add(new LicenseSourceFilePath(Path.Combine(Environment.GetFolderPath(Environment.SpecialFolder.LocalApplicationData), "ParticularSoftware", "license.xml")));
+            sources.Add(new LicenseSourceFilePath(Path.Combine(AppDomain.CurrentDomain.BaseDirectory, "license.xml")));
```

Comment:
the documentation sometimes mentions a "License.xml" file. Is `LicenseSourceFilePath` able to handle case-sensitivity internally or do we need to add multiple sources therefore (at least for linx/macos)?